ls1intum/Artemis
# `Plagiarism`: Rework data model

### Checklist
#### General


#### Server

### Motivation and Context


The current data model of the plagiarism comparisons is not suitable for the intended functionality and caused a lot of issues, especially in regard to the load on the server.

### Description

This pull request prepares the data model for the rework of the plagiarism management. The new entities and necessary changes can also be seen in the screenshot attached below.

### Steps to Test
Start the Artemis server locally and check if the database changes have been successfully been made as described in the migration file. In order to do this simply connect to your local database via your favourite tool. 

### Review Progress





#### Code Review

### Screenshots


<img width="1141" alt="Bildschirmfoto 2022-02-11 um 09 43 45" src="https://user-images.githubusercontent.com/51987021/159786178-a2ebe0a0-91ce-4730-8d4c-dcf2116fb6d8.png">
